src/es/components/organisms/grid/12er-/12er-.css.js

```diff
@@ -0,0 +1,46 @@
+/* css */ `
+:host > section {
+  grid-template-columns: repeat(12, 1fr);
+  gap: ${this.getAttribute('gap') || 'var(--grid-12er-grid-gap, 1rem)'};
+}
+/* col-lg */
+${Array.from(this.section.querySelectorAll('[col-lg]')).reduce((acc, node) => acc + (acc.includes(`[col-lg="${node.getAttribute('col-lg')}"]`)
+  ? ''
+  : `
+    :host > section >*[col-lg="${node.getAttribute('col-lg')}"] {
+      grid-column: span ${node.getAttribute('col-lg') || node.getAttribute('col-md') || '12'};
+    }
+  `),
+'')}
+
+/* col-md */
+@media only screen and (max-width: ${this.getAttribute('tablet-breakpoint') || '1024px'}) {
```

Comment:
could also use the mobile break point from shadow.js, which covers all common desktop/mobile breakpoints in the project